components/activity_and_insights/insights/top_reactions/top_reactions_table/top_reactions_table.tsx

```diff
@@ -0,0 +1,144 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+import React, {memo} from 'react';
+
+import {FormattedMessage} from 'react-intl';
+
+import DataGrid, {Row, Column} from 'components/admin_console/data_grid/data_grid';
+import RenderEmoji from 'components/emoji/render_emoji';
+
+import './../../../activity_and_insights.scss';
+
+const TopReactionsTable = () => {
+    const getColumns = (): Column[] => {
+        const columns: Column[] = [
+            {
+                name: (
+                    <FormattedMessage
+                        id='insights.topReactions.rank'
+                        defaultMessage='Rank'
+                    />
+                ),
+                field: 'rank',
+                className: 'rankCell',
+                width: 0.2,
+            },
+            {
+                name: (
+                    <FormattedMessage
+                        id='insights.topReactions.reaction'
+                        defaultMessage='Reaction'
+                    />
+                ),
+                field: 'reaction',
+            },
+            {
+                name: (
+                    <FormattedMessage
+                        id='insights.topReactions.timesUsed'
+                        defaultMessage='Times used'
+                    />
+                ),
+                field: 'times_used',
+            },
+        ];
+        return columns;
+    };
+
+    const getRows = (): Row[] => {
+        return [
+            {
+                cells: {
+                    rank: (
+                        <span className='cell-text'>
+                            {'1'}
+                        </span>
+                    ),
+                    reaction: (
+                        <>
+                            <RenderEmoji
+                                emojiName={'grinning'}
+                                size={24}
+                            />
+                            <span className='cell-text'>
+                                {'grinning'}
```

Comment:
These are hard-coded temporarily until the data wire-up?

Reply:
Yep